sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/Gallery.cs

```diff
@@ -74,15 +75,11 @@ public Gallery(string location, string id = default(string), string name = defau
         public GalleryIdentifier Identifier { get; set; }
 
         /// <summary>
-        /// Gets the current state of the gallery.
+        /// Gets or sets possible values include: 'Creating', 'Updating',
+        /// 'Failed', 'Succeeded', 'Deleting', 'Migrating'
         /// </summary>
-        /// <remarks>
-        /// The provisioning state, which only appears in the response.
-        /// Possible values include: 'Creating', 'Updating', 'Failed',
-        /// 'Succeeded', 'Deleting', 'Migrating'
-        /// </remarks>
         [JsonProperty(PropertyName = "properties.provisioningState")]
-        public string ProvisioningState { get; private set; }
```

Comment:
This removal of Private seems incorrect. The Swagger [PR ](https://github.com/Azure/azure-rest-api-specs/pull/18680/files#diff-2861a3a31f910fbcb3cbc0af6f313527001ff65e8360cfcc566ae280a27b8bf9R2887)is not removing the ReadOnly: True tag, so this should still be ReadOnly.

Reply:
This issue occurs in multiple files in this PR.

Reply:
Might need to keep the ReadOnly:True tag directly in the ProvisioningState property instead of the model. Try that.

Reply:
this is resolved